src/l500/ac-trigger.cpp

```diff
@@ -377,6 +377,7 @@ namespace ivcam2 {
 
     ac_trigger::~ac_trigger() 
     { 
+        reset();
```

Comment:
Why reset()? Just to set _is_processing to false?

Reply:
The "reset" function also releases the frames so I figured it can't be bad to use it.

Do you prefer just setting _is_processing to false?

Reply:
My only problem with reset() is that it actually writes something to the log.
I think if you want something specific, make it specific in the dtor.

Reply:
Removed call and replace with _is_processing = false + a log message